File: Pytonn/Driver.py
from account import account
class Driver(account):
    def __init__(self, name, document, email, password):
        super().__init__(name, document, email, password)

    def printDataUser(self):
        print("Hola "+self.name+",bienvenido de vuelta!\nNumero de documento: "+self.document)

File: Java/Driver.java
package Java;

class Driver extends account{
    public Driver(String name,String document,String email,String password){
        super(name,document,email,password);
    }
    void printDataDriver(){
        System.out.println("Hola "+name+",bienvenido de vuelta!\nNumero de documento: "+document);
    }
}
